packages/fetch-engine/src/download.ts

```diff
@@ -455,9 +459,10 @@ async function saveFileToCache(
   }
 }
 
-function engineTypeToBinaryType(engineType: string, binaryTarget: string): string {
-  if (BinaryType[engineType]) {
-    return BinaryType[engineType]
+// TODO ???? What is this doing?
+function barToBaz(engineType: string, binaryTarget: string): string {
```

Comment:
Looks like it's returning the "file name" of the engine, and it handles "native" 
@janpio Can you revert this rename? "barToBaz"